src/commands/start.js

```diff
@@ -66,21 +65,24 @@ async function start(opts: *) {
   // Run `adb reverse` on Android
   if (opts.platform === 'android') {
     const args = `reverse tcp:${opts.port} tcp:${opts.port}`;
-    const adb = process.env.ANDROID_HOME
-      ? `${process.env.ANDROID_HOME}/platform-tools/adb`
-      : 'adb';
+    /**
+     * ADB should be taken from path (the correct setup is even in React Native documentation)
```

Comment:
This comment makes sense in the context of the PR, but won't make sense after merged, where else you should take the command from if not path? :p

Let's remove this command and `inline` the `adb` variable.

Reply:
OK. I wanted to be explicit for next generations XD